### ISSUE TITLE
/emojicloud does not work; remains stuck on "Generating EmojiCloud for [target]"

### ISSUE THREAD

**Issue by adamnwerbik** [olliequ/KittyBot#59]
self-explanatory
![image](https://user-images.githubusercontent.com/69626130/202705708-8c589b4a-b4ed-44f0-9665-c6a150884895.png)

**Comment by a-lexer** [comment]
have you tried restarting

**Comment by flukiluke** [comment]
Error observed in production:
```
Nov 18 23:27:22 plum kitti[522372]: E 2022-11-18 12:27:22,743 hikari.event_manager: an exception occurred handling an event (SlashCommandErrorEvent)
Nov 18 23:27:22 plum kitti[522372]: Traceback (most recent call last):
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/lightbulb/app.py", line 1154, in invoke_application_command
Nov 18 23:27:22 plum kitti[522372]:     await context.invoke()
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/lightbulb/context/base.py", line 292, in invoke
Nov 18 23:27:22 plum kitti[522372]:     await self.command.invoke(self)
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/lightbulb/commands/base.py", line 544, in invoke
Nov 18 23:27:22 plum kitti[522372]:     await self(context, **kwargs)
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/lightbulb/commands/base.py", line 459, in __call__
Nov 18 23:27:22 plum kitti[522372]:     return await self.callback(context, **kwargs)
Nov 18 23:27:22 plum kitti[522372]:   File "/home/extensions/emojicloud.py", line 38, in main
Nov 18 23:27:22 plum kitti[522372]:     await emoji_cache.download_emoji(i[0], ctx.bot)
Nov 18 23:27:22 plum kitti[522372]:   File "/home/emoji_cache.py", line 32, in download_emoji
Nov 18 23:27:22 plum kitti[522372]:     emoji = await bot.rest.fetch_emoji(int(os.environ["DEFAULT_GUILDS"].split(',')[0]), emoji_id)
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/hikari/impl/rest.py", line 2128, in fetch_emoji
Nov 18 23:27:22 plum kitti[522372]:     response = await self._request(route)
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/hikari/impl/rest.py", line 817, in _request
Nov 18 23:27:22 plum kitti[522372]:     await self._handle_error_response(response)
Nov 18 23:27:22 plum kitti[522372]:   File "/usr/local/lib/python3.10/site-packages/hikari/impl/rest.py", line 839, in _handle_error_response
Nov 18 23:27:22 plum kitti[522372]:     raise await net.generate_error_response(response)
Nov 18 23:27:22 plum kitti[522372]: hikari.errors.NotFoundError: Not Found 404: (10014) 'Unknown Emoji' for https://discord.com/api/v8/guilds/813213508036067348/emojis/813233764213850173
Nov 18 23:27:22 plum kitti[522372]: The above exception was the direct cause of the following exception:
Nov 18 23:27:22 plum kitti[522372]: Traceback (most recent call last):
Nov 18 23:27:22 plum kitti[522372]:   File "/home/bot.py", line 31, in on_error
Nov 18 23:27:22 plum kitti[522372]:     raise event.exception
Nov 18 23:27:22 plum kitti[522372]: lightbulb.errors.CommandInvocationError: An error occurred during command 'emojicloud' invocation
```

**Comment by dakshAg** [comment]
Fixed in #61
